src/actions/infoAction.ts

```diff
@@ -7,6 +7,10 @@ export default function (): void {
   info('=========*** Info ***==========\n');
 
   const { market, provider, network, wallet } = state.get();
+  if (!wallet) throw new Error('wallet is undefined');
```

Comment:
I think info should show low level what we got. so we can know wich thins is missing or not